### DISCUSSION TITLE
Split Audio into multiple audio files according to a recognised phrase

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#431] [Q&A]
Example: audio is 40 mins
Speaker says split at intervals of 10 minutes.
Whisper recognises the word "split" and then splits the audio at the point the speaker said "split" and then makes the 0:00-10:00 of the file into a new audio file.
So basically in this example the 40 min audio would be split into 4 different files at the places where the speaker said "split".
Anyone know how to do this?

**Comment by Qualzz** [comment]
If you transcribe the whole audio, whisper will provide you with a timed transcription. Maybe you can simply look when those "split" words appear in the transcription, and use the time at the moment it was said. You should then know every moment where the word split way said and can then cut your audio segment into those parts.
